packages/std/src/errors/std_error.rs

```diff
@@ -468,7 +468,8 @@ pub enum DivisionError {
     #[error("Overflow in division")]
     Overflow,
 }
-
+/// Errors that occur when multiplying a value by a ratio in a checked manner.
```

Comment:
This says "ratio", but this one is actually specific to multiplications with types that implement the `Fraction` type.
```suggestion
/// Errors that occur when multiplying a value by a [Fraction](crate::Fraction) in a checked manner.
```